src/actions/contract.js

```diff
@@ -39,15 +41,15 @@ function getContractParams({
   descrPath, contractAddress, contractSource, contractBytecode, contractAci,
 }, { dummySource } = {}) {
   if (descrPath && fs.existsSync(resolve(descrPath))) {
-    const { address, ...other } = JSON.parse(readFile(descrPath));
+    const { address, ...other } = JSON.parse(readFile(descrPath).toString());
```

Comment:
this shouldn't need a `.toString()`, same for occurences below

Reply:
`readFile` returns Buffer, `JSON.parse` accepts a string, while this working because of implicit conversion to string, I would add an explicit one to fix a warning in my IDE

In TS this also would be an error:

> error TS2345: Argument of type 'Buffer' is not assignable to parameter of type 'string'.
console.log(JSON.parse(buffer))